app/assets/javascripts/miq_explorer.js

```diff
@@ -341,5 +347,7 @@ ManageIQ.explorer.processReplaceRightCell = function(data) {
     miqSparkleOff();
   }
 
+  ManageIQ.explorer.resetReduxForms();
```

Comment:
@rvsia This is not the right place..  `ManageIQ.explorer.process` is the main entrypoint, `processReplaceRightCell` is only one possible branch.. `processFlash` and `processReplaceMainDiv` also handle partial updates.

(But you mentioned you'll deal with this more in a follow-up PR, and this won't break anything as is, so ignoring.)

Reply:
`resetReduxForms` is also in `processReplaceMainDiv`. I didn't want to add this function to `processFlash` as I thought that it manages the flash messages (or does it do something else?) and I don't want to reset the forms when messages change. Right now, all forms are in `main_div` or in `right_cell`, so it is a sufficient (and I suppose all forms would be there).

Reply:
Aaah, sorry, I missed it in processReplaceMainDiv, you're right :)